cmd/kubectl_token.go

```diff
@@ -402,15 +408,17 @@ func basicAuth(input *LoginInput) (managementClient.Token, error) {
 			"[%v] message:[%v]", apiError["code"], apiError["message"])
 	}
 
-	err = json.Unmarshal(response, &token)
+	err = json.Unmarshal(respBody, &token)
 	if err != nil {
 		return token, err
```

Comment:
Ditto: Is it ok to return both error and token here ? Should it be the error alone instead, without the token ?

Reply:
See above.